src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -932,6 +932,77 @@ class CreateTableStatementSegment(BaseSegment):
     )
 
 
+@postgres_dialect.segment()
+class CreateTableAsStatementSegment(BaseSegment):
+    """A `CREATE TABLE AS` statement.
+
+    As specified in https://www.postgresql.org/docs/13/sql-createtableas.html
+    """
+
+    type = "create_table_as_statement"
+
+    match_grammar = Sequence(
+        "CREATE",
+        OneOf(
+            Sequence(
+                OneOf("GLOBAL", "LOCAL", optional=True),
+                Ref("TemporaryGrammar", optional=True),
+            ),
+            "UNLOGGED",
+            optional=True,
+        ),
+        "TABLE",
+        Ref("IfNotExistsGrammar", optional=True),
+        Ref("TableReferenceSegment"),
+        AnyNumberOf(
+            Sequence(
+                Bracketed(
+                    Delimited(
+                        Sequence(
+                            Ref("ColumnReferenceSegment"),
+                            AnyNumberOf(Ref("ColumnConstraintSegment")),
```

Comment:
I don't think CREATE TABLE AS supports column constraints, just renaming of the column name

Reply:
Agree. This has been changed.